version/src/lib.rs

```diff
@@ -64,10 +85,10 @@ pub mod error {
                     "Cannot encode unsupported version from SCALE to JSON"
                 }
                 #[cfg(feature = "json")]
-                Self::Serde(_) => "JSON (de)serialization issue",
```

Comment:
I suppose we could add the details back when we actually start doing something with them.